app/views/course_templates/index.html.erb

```diff
@@ -1,14 +1,14 @@
-<h1>Listing course_templates</h1>
+<h1>Course templates</h1>
 
-<table>
+<table class="table table-striped table-condensed">
   <thead>
     <tr>
       <th>Name</th>
       <th>Title</th>
       <th>Description</th>
-      <th>Material url</th>
-      <th>Source url</th>
-      <th>Expires at</th>
+      <th>Material&nbsp;url</th>
+      <th>Source&nbsp;url</th>
+      <th>Expires&nbsp;at</th>
```

Comment:
Might be easier to add some CSS that makes all `<th>` not wrap. Not sure if that's good for *all* our tables though (I'd check especially the points tables when large).